include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -3195,10 +3225,12 @@ FEEvaluationBase<dim, n_components_, Number, is_face, VectorizedArrayType>::
   read_cell_data(const AlignedVector<VectorizedArrayType> &array) const
 {
   VectorizedArrayType out = Number(1.);
-  internal::process_cell_data(
-    *this, this->matrix_free, array, out, [](auto &local, const auto &global) {
-      local = global;
-    });
+  internal::process_cell_or_face_data(
+    *this,
+    array,
+    out,
+    [](auto &local, const auto &global) { local = global; },
+    true);
```

Comment:
If done as above, we could now write
```suggestion
    this->get_cell_ids(),
    array,
    out,
    [](auto &local, const auto &global) { local = global; });
```